catalyst/utils/metrics/accuracy.py

```diff
@@ -13,27 +13,62 @@
 from catalyst.utils.torch import get_activation_fn
 
 
+def multi_class_accuracy(
+    outputs: torch.Tensor, targets: torch.Tensor, topk: Tuple = (1,),
+) -> List:
+    max_k = max(topk)
+    batch_size = targets.size(0)
+
+    if len(outputs.shape) == 1 or outputs.shape[1] == 1:
+        # binary accuracy
+        pred = outputs.t()
+    else:
+        # multi-class accuracy
+        _, pred = outputs.topk(max_k, 1, True, True)  # noqa: WPS425
+        pred = pred.t()
+    correct = pred.eq(targets.long().view(1, -1).expand_as(pred))
+
+    output = []
+    for k in topk:
+        correct_k = correct[:k].view(-1).float().sum(0, keepdim=True)
+        output.append(correct_k.mul_(1.0 / batch_size))
+    return output
+
+
+def multi_label_accuracy(
+    outputs: torch.Tensor,
+    targets: torch.Tensor,
+    threshold: float = torch.Tensor,
+) -> torch.Tensor:
+    outputs = (outputs > threshold).long()
+    output = (targets.long() == outputs.long()).sum().float() / np.prod(
```

Comment:
We already converted outputs to `long` type in the line above.
```suggestion
    output = (targets.long() == outputs).sum().float() / np.prod(
```